linux/just_files/singularity_functions.bsh

```diff
@@ -113,6 +122,7 @@ function singularity_env_pass()
 #           * ``singular_environment`` - A key-value pair array of environment variables that need to be passed to the singularity container.
 #           * ``singular_image`` - The filename of the singularity image used.
 #           * ``singular_mount_points`` - List of mount points that need to be created when importing a docker image to singularity. This is only important for older systems that do not support overlayfs (like CentOS 6). Directories must end in ``/``, and files must not.
+#           * ``singular_instance_name`` - The name of the singularity instance uses, including a unique hash for the project (Based off of username and path)
```

Comment:
The name the singularity instance uses,
or 
The name of the singularity instance,

Reply:
The first one, because "singularity instance" == "docker compose service", for the most part, so it makes more sense